elastic/kibana
# [Synthetics] Refactor code to use monitor location id for filtering

## Summary

Use monitor location id for filtering. 

Also removed redundant code which was their to account for missing label in location objects which were part of monitors.

### Testing

1. Make sure location filtering is working as expected in overview and management pages
2. Make sure sorting by locaiton is working as expected on both pages
